monkeys/clique/deployments/clique.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: chaos-clique-first
```

Comment:
This needs a cooler name. You are even allowed to make a pin in this case. Use this opportunity wisely.